README.md

```diff
@@ -107,9 +107,11 @@ clasp deployments # List all deployment IDs
 ### Open the Project on script.google.com
 
 ```
-clasp open
+clasp open [scriptId]
 ```
 
+Takes an optional argument of the scriptId that you'd like to open. Else, opens script from `.clasp.json`
```

Comment:
First describe what it does without options (90% of cases). Then describe options.
```
Opens the `clasp` project on script.google.com. Provide a `scriptId` to open a different script.
```